src/main/java/com/tournament/cup/play/Play.java

```diff
@@ -0,0 +1,16 @@
+package com.tournament.cup.play;
+
+import lombok.Data;
+
+@Data
+public class Play {
+    private Squad squadFirst;
```

Comment:
Better to name it `firstSquad`, `secondSquad`.
It sounds more native in English.